### PR TITLE
Add ansible_collections_path in job invocation advanced_fields

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Dependant PR https://github.com/SatelliteQE/robottelo/pull/15595

### CODE REVIEW

**Review by LadislavVasina1 — APPROVED** [review]

File: airgun/views/job_invocation.py [hunk 1]
```diff
@@ -56,9 +56,7 @@ class target_hosts_and_inputs(WizardStepView):
 
         targetting_type = Select(locator='//div[button[@aria-haspopup="listbox"]]')
         # https://bugzilla.redhat.com/show_bug.cgi?id=2265127
```
Comment (shweta83) [inline airgun/views/job_invocation.py:58]: The bug has been fixed by Dev but yet to be verified by QE so we can verify(using airgun) and remove this bugzilla. Just a non-blocker comment.

**Review by shweta83 — APPROVED** [review]
Ack. Pending PRT passing with related robottelo PR.

File: airgun/views/job_invocation.py (hunk 1 above)
Comment (Gauravtalreja1) [reply airgun/views/job_invocation.py:58]: Same I'm discussing here https://github.com/SatelliteQE/airgun/pull/1464#discussion_r1671249484, we can rebase and merge this afterwards

Comment (shweta83) [reply airgun/views/job_invocation.py:58]: Sure.

**Review by sambible — APPROVED** [review]

**Comment by Gauravtalreja1** [comment]
PRT passed on https://github.com/SatelliteQE/robottelo/pull/15595

**Review by shweta83 — APPROVED** [review]
Ack

**Pull request merged**